docs/Getting Started.md

```diff
@@ -90,7 +92,17 @@ Cloudflow offers several sbt plugins that abstract quite a bit of boilerplates n
 
 > **Note:** You can use multiple plugins to develop an application that uses multiple runtimes (Akka, Spark, Flink etc.). For simplicity of this example we will be using only one.
 
-The above build script is standard Scala sbt - the only difference is the plugin which we provide as part of Cloudflow.
+The above build script is standard Scala sbt - the only difference is the plugin which we provide as part of Cloudflow. And here's how we specify the name and version of the plugin to use:
+
+**project/cloudflow-plugins.sbt**
+
+```
+// Resolver for the cloudflow-sbt plugin
+//
+resolvers += "Akka Snapshots" at "https://repo.akka.io/snapshots/"
```

Comment:
@debasishg this resolver is not required for resolving the sbt plugin. AFAIK this esolver is not required for any part of Cloudflow and should be removed where we still see it.

Background: a long time ago we added some features to Akka Streams and while those features were not released yet we used Akka snapshots.

Reply:
ok - will remove it ..